Products/PrintingMailHost/__init__.py

```diff
@@ -21,7 +24,8 @@ def initialize(context):
 
     # check to see if the environment var is set to a 'true' value
     if (ENABLED is not None and ENABLED.lower() in TRUISMS) or \
-       (ENABLED is None and DevelopmentMode is True):
+       (ENABLED is None and getConfiguration().debug_mode is True):
         LOG.warning("Hold on to your hats folks, I'm a-patchin'")
-        import Patch
+        DevelopmentMode = True  # noqa
```

Comment:
setting DevelopmentMode to True here might be misleading since it is also sufficient to set the environment variable `ENABLE_PRINTING_MAILHOST` to `True` to enter this condition.

i'd rather skip this here. afaict the variable `DevelopmentMode` is not used in this package at all.
if other packages depend on that they'd need to be fixed.

Reply:
Agreed, in current master we only import `DevelopmentMode` to check it once at startup for true/false, but we do not need it afterwards. So the variable can be removed.
No need to keep it for backwards compatibility, I would say.